package.json

```diff
@@ -182,7 +182,8 @@
     "web3": "^0.20.7",
     "web3-stream-provider": "^4.0.0",
     "webrtc-adapter": "^6.3.0",
-    "xtend": "^4.0.1"
+    "xtend": "^4.0.1",
+    "graphql-request": "^1.8.2"
```

Comment:
Where is this dependency used?

Reply:
Thats an unused dependency will remove it.